components/ReviewCard.tsx

```diff
@@ -100,48 +100,44 @@ export const ReviewCard: React.FC<ReviewCardProps> = ({ submissionData }) => {
   }
 
   return (
-    <>
-      {diff && (
```

Comment:
I deleted this check since it created an untestable branch. Local component `DiffView` already has default case when no diff is provided but since it's called as part of this expression you can't access it.

Reply:
Will this change affect the front end user experience?

Reply:
It's from an old commit, current pr added one extra export instead of changing component. It should not affect end use either way.